counterOutput.js

```diff
@@ -0,0 +1,26 @@
+export default class CounterOutput {
+  constructor(labelText, padding=1) {
+    this.node = document.createElement("div");
+    this.label = document.createElement("div");
+    this.counter = document.createElement("div");
+    this.padding = padding;
+    this._init(labelText);
+  }
+
+  _init(labelText) {
+    this.node.className = "counter-output-container";
+    this.label.append(labelText);
```

Comment:
I have changed it to `labelNode`. The reason for using `.append()` is because it accepts **both** DOMString objects and node objects. This gives us the flexibility to insert a text as the label or a node as a label. In the case of the cells display, the `labelNode` is displaying a node `colourBox`.

Reply:
that sounds good. Also, `_init` is a weird name here. When I see `init` I ask myself why even have this function when the constructor does initing. If it does something else the name should indicate that.